src/assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -84,14 +84,14 @@ fieldset li {
 
 .mzp-c-fieldnote {
     @include text-body-sm;
-    color: $color-gray-80;
+    color: $color-dark-gray-40;
     display: block;
     font-weight: normal;
 }
 
 .mzp-c-form-errors {
-    background: $color-yellow-50;
-    border: 2px solid $color-yellow-60;
+    background: $color-yellow-05;
+    border: 2px solid $color-yellow-30;
```

Comment:
Did the red and yellow colours change?

Reply:
I changed them, but maybe this PR isn't the place to do that and we should just match colors as closely as possible now and make the colors better with the other form work.

Before:
![image](https://user-images.githubusercontent.com/205591/72463562-5b982900-3788-11ea-8db7-afa8f0e82a9f.png)

After:
![image](https://user-images.githubusercontent.com/205591/72463531-4a4f1c80-3788-11ea-852e-05f874384999.png)

Reply:
If the red and yellow changed and the contrast is poor now it's okay to change them. And if that's the case we should provide guidance for people updating.

If the red and yellow tokens did not change, yeah, we should leave the UI tweaks for a different PR.